super_dpf.py

```diff
@@ -0,0 +1,129 @@
+import json
+import os
+import sys
+import collections
+
+
+class SuperDPF(object):
+    PATHS = {'project': '/home/pi/SuperDPF'}
```

Comment:
You can get the username with getpass.getuser() or the full path with os.path.expanduser('~/SuperDPF')
This would make the project portable between users.